docs/pages/Getting_Started.md

```diff
@@ -2258,8 +2279,26 @@ obs_def modules (for example, `obs_def_1d_state_mod.f90` or
 files are included in the preprocessor namelist, a minimal `obs_def_mod.f90` is
 created which can only support identity forward observation operators.
 
-To add a new observation type which does not fit into any of the
-already-defined obs_def files, a new file should be created in the
+<span id="newQUANTITIES" class="anchor"></span> [](#newQUANTITIES)
+#### To add a new *quantity*,
+
+the `DARTHOME/assimilation_code/modules/observations/DEFAULT_obs_kind_mod.F90` file
+must be edited to include a unique integer identifier for the quantity.
+There are two, sometimes three places in the file that require modification.
+The first is in the table of existing NAME-value pairs and the second is perhaps
+halfway down the file in the section after:
+
+~~~
+! DART PREPROCESS OBS_QTY_INFO INSERTED HERE
+~~~
+
+If you add an observation quantity with an integer value greater than **max_defined_quantities**,
+make sure you increase *max_defined_quantities*.  
+
+<span id="newTYPES" class="anchor"></span> [](#newTYPES)
+#### To add a new observation *type* which does not fit into any of the already-defined obs_def files
+
+a new file should be created in the
```

Comment:
Same comment as above - the style of subheaders should stay the same as the rest of the document.

Reply:
Made me think a little ... I added a paragraph to explain what should happen if the new observation type should be added to an existing obs_def module. to wit:

The observation support modules are in the `DARTHOME/observations/forward_operators` directory.
If the observation type logically belongs in an existing observation module, it should be 
fairly obvious how to add support for a new observation type. Feel free to use ideas from other modules - look around!

Reply:
do we say anywhere here that for observation *types* that this is where they are defined.  for observation *quantities*, they have to already exist in the obs_kind_mod.

Reply:
Yes ... the section immediately before the observation types is all about quantities, 

check out:

https://github.com/timhoar/DART/blob/getting_started_page/docs/pages/Getting_Started.md#newQUANTITIES

Reply:
And in the paragraph of adding observation types (which appears AFTER the observation quantity section):   _If the new observation type also requires a new observation quantity, there is more work to be done._